cmd/hamsible/export/cmd_util.go

```diff
@@ -0,0 +1,29 @@
+package export
+
+import (
+	"fmt"
+	"strings"
+)
+
+//use this function for parsing arguments
+func parse(args []string) error {
+	if len(args) != 1 {
+		return fmt.Errorf("please specify role name")
+	}
+	roleName = args[0]
+	if len(roleName) == 0 {
+		return fmt.Errorf("role name should not be empty")
+	}
+
+	return nil
+}
+
+func verifyFlags() error {
+	if len(helmChartRef) == 0 {
+		return fmt.Errorf("Please Specify Helm Chart path")
+	}
+	if strings.ContainsAny(helmChartRef, " ") {
+		return fmt.Errorf("Helm Chart path Contain Spaces")
```

Comment:
We should probably check for the path existence here.  You could make use of something similar to what I put in convert.go:

```
// Checks a directory for existence, exiting fatally if said directory does not exist.
func checkDirectoryExistence(directory string, errorMessage string) {
	if _, err := os.Stat(directory); os.IsNotExist(err) {
		logrus.Errorf("%s: %s", errorMessage, directory)
		logrus.Fatal(err)
	}
}
```